### PR TITLE
add scripts and instructions on how users can monitor their sync apps

### PR DESCRIPTION
**Pull request by wei-lee**
To verify:

1. Provision the data sync app into a namespace
2. Follow the instructions in the readme here and at the end you should see a syn dashboard running

### CODE REVIEW

**Comment by wei-lee** [comment]
ping @wtrocki @darahayes @StephenCoady

**Comment by wtrocki** [comment]
Working on it.

File: openshift/metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Usage
+
+1. Make sure you have `oc` installed and targeted the namespace where the sync app is deployed.
+2. Run the script to provision Prometheus and Grafana:
+    ```
+    ./install-metrics.sh
+    ```
+3. Add the following annotation to the `Deployment` or `DeploymentConfig` of your sync apps (change the value for the endpoint to match the actual metrics endpoint if it's not `/metrics`):
```
Comment (wtrocki) [inline openshift/metrics/README.md]: I'm just wondering. I might not get that right. Can we put this into the template and ask people to edit when endpoint is different?

File: openshift/metrics/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    # If it's HTTP, use this annotation
+    org.aerogear.metrics/plain_endpoint: /metrics
+    # Or if it's HTTPS, use this one
+    org.aerogear.metrics/ssl_endpoint: /metrics
+    ```
+
+4. Add the following label to the `Deployment` or `DeploymentConfig` of your sync apps:
+
+   ```
+   datasync: 'yes'
```
Comment (wtrocki) [inline openshift/metrics/README.md]: [Question] Unrelated to PR. Is that trigger metrics operator to include dashboards?

**Comment by wtrocki** [comment]
Executed script

```
s ‹add-metrics›
╰─$ ./install-metrics.sh
Generating Prometheus configuration file
Provisioning Prometheus
secret/prom created
secret/prom-alerts created
role.rbac.authorization.k8s.io/monitor-service created
serviceaccount/prom created
rolebinding.rbac.authorization.k8s.io/prometheus-monitoring created
route.route.openshift.io/prom created
service/prom created
service/prometheus-internal created
statefulset.apps/prom created
route.route.openshift.io/prom-alerts created
service/prom-alerts created
Prometheus provisioned
Provisioning Grafana
configmap/grafana-config created
configmap/grafana-datasources created
configmap/grafana-providers created
configmap/grafana-dashboards created
serviceaccount/grafana created
route.route.openshift.io/grafana created
service/grafana created
persistentvolumeclaim/grafana-data-claim created
deploymentconfig.apps.openshift.io/grafana created
Grafana Provisioned
```

Added label. 
It works!

**Review by wtrocki — APPROVED** [review]

**Comment by aliok** [comment]
@wei-lee just saw this open. Can we merge this?

**Comment by wtrocki** [comment]
I have verified all code. It should ok to merge

**Comment by wei-lee** [comment]
@aliok @wtrocki I made a small change to the PR to make it a bit easier to use. Mind taking another look before I merge? Thanks.

**Comment by wtrocki** [comment]
Added script works! Good to merge

**Review by wtrocki — APPROVED** [review]

**Pull request merged**